README.md

```
@@ -1,237 +1,99 @@
+The [Armada documentation](https://armadaos.dev/) is the source of truth for
+device support, installation, updates, current limitations, and recovery. Use
+the guides there rather than instructions copied from older releases or posts.
+
+| I want to… | Guide |
+|---|---|
+| Install Armada | [Flash to an SD card](https://armadaos.dev/getting-started/flashing-to-an-sd-card/) |
+| Check my handheld | [Supported devices](https://armadaos.dev/devices/supported-devices/) |
+| Learn the interface | [Using Armada](https://armadaos.dev/using-armada/) |
+| Update an installation | [Updating](https://armadaos.dev/getting-started/updating/) |
+| Find help | [FAQ](https://armadaos.dev/troubleshooting/frequently-asked-questions/) · [Known issues](https://armadaos.dev/troubleshooting/known-issues/) |
+| Report a bug | [Github Issues](https://github.com/armada-os/armada/issues)
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)